src/data/DataTypes.ts

```diff
@@ -29,7 +29,11 @@ export interface Technique {
   file_score: number;
   cloud_score: number;
   hardware_score: number;
-  actionability_score: number;
+  actionability_score: {
```

Comment:
Just to avoid updating things like this in two places (`Technique` and `ExportedTechnique`). Let's consider breaking these out into sub-interfaces. Something like:
```ts
interface TechniqueBase {
  tid: string;
  name: string;
  description: string;
  url: string;
  detection: string;
  mitigations: Array<Mitigation>;
  subtechniques: Array<Subtechnique>;
}

interface TechniqueCoverage {
  process_coverage: boolean;
  network_coverage: boolean;
  file_coverage: boolean;
  cloud_coverage: boolean;
  hardware_coverage: boolean;
}

interface TechniqueScores {
  score: number;
  process_score: number;
  network_score: number;
  file_score: number;
  cloud_score: number;
  hardware_score: number;
  actionability_score: {
    combined_score: number;
    mitigation_score: number;
    detection_score: number;
  };
  choke_point_score: number;
  prevalence_score: number;
}
```
etc, etc.

Then, the sub-interfaces can be clicked together into the `Technique` and `ExportedTechnique` types:

```ts
export Technique extends TechniqueBase, TechniqueCoverage, TechniqueScores {
   ...
}

export ExportedTechnique extends TechniqueBase, TechniqueScores {
  rank: number;
  ...
}
```